src/traces/scatterternary/plot.js

```diff
@@ -13,11 +13,16 @@ var scatterPlot = require('../scatter/plot');
 
 
 module.exports = function plot(ternary, data) {
+    var plotContainer = ternary.plotContainer;
+
+    // remove all nodes inside the scatter layer
+    plotContainer.select('.scatterlayer').selectAll('*').remove();
```

Comment:
I remember using `html('')` at some point, but switch to `selectAll('*').remove()` because it wasn't  properly in IE.